util/dvsim/SimCfg.py

```diff
@@ -113,6 +113,15 @@ def __init__(self, flow_cfg_file, proj_root, args):
         self.parse_flow_cfg(flow_cfg_file)
         self._post_parse_flow_cfg()
 
+        # If self.tool is None at this point, there was no --tool argument on
```

Comment:
I think this can be moved inside the `if` clause on 149.

Reply:
Heh, that's funny - I didn't realise that this duplicates some code in PR #2141. I'll move this too, but one of PRs is going to need tidying up, depending on which gets merged first.

Reply:
I have tested this PR. It will likely be merged before the other one.